webvtt.go

```diff
@@ -47,7 +47,8 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 	// Skip the header
 	for scanner.Scan() {
 		line = scanner.Text()
-		if len(line) > 0 && line != "WEBVTT" {
+		line = strings.Trim(line, "\xef\xbb\xbf") // strip BOM chars
```

Comment:
Could you use the `strings.TrimPrefix()` method and the `BOMHeader` constant instead?